lib/atmosphere/clockwork.rb

```diff
@@ -29,17 +25,27 @@ module Clockwork
         Atmosphere::BillingWorker.perform_async
       end
 
-      every(Atmosphere.url_monitoring.pending.seconds, 'monitoring.http_mappings.pending') do
+      every(Atmosphere.url_monitoring.pending.seconds,
+            'monitoring.http_mappings.pending') do
```

Comment:
Here yes. The main difference is lines 7-8, 12-13 and 43-48. The rest is just formatting